### ISSUE TITLE
USING THIS SETUP ON CPU

### ISSUE THREAD

**Issue by shanky100** [saharmor/whisper-playground#42]
Hi, just wanted to ask if it is possible to use this  repo using CPU , if yes then do I need to make any specific changes to the code or any other additional step. OS: Ubuntu 20 LTS and python 3.8

**Comment by shanky100** [comment]
I managed to get the system up and running, whatever I speak the analog signal changes but I do not received any transcription on the UI/web page.  I cna see the transcription being generated in the terminal logs.

Is someone else facing the same issue, any suggestions will be helpful.

**Comment by saharmor** [comment]
Can you attach the console log (a screenshot would do) + a screen recording?

**Comment by ethanzrd** [comment]
Following Sahar's request, please make sure that the screen recording is at least 15 seconds if you're using the real-time mode with a transcription timeout of 5 seconds :)

**Comment by shanky100** [comment]
https://github.com/saharmor/whisper-playground/assets/132049696/d4355b4c-8e27-422f-b345-b76e69bf1f87
@ethanzrd  I just checked what you suggested but I don't see any temp folder getting created in the backed folder.
I by mistake said backed folder in the video but I mean temp folder only. YOu can check the recording

**Comment by ethanzrd** [comment]
You're right, I was mistaken. This only applies to the sequential mode. Speaking of which, could you try it and let me know if you're having any problems there? If you are, then the batch.wav file will be present for you to include :)

**Comment by shanky100** [comment]
@ethanzrd I checked with sequential mode. There it generated the wav file and it is having the voice recording. But I neither see any transcription in my terminal logs nor on the whisper playground WEB PAGE.

I am sharing the wav file.
[temp.zip](https://github.com/saharmor/whisper-playground/files/12410805/temp.zip)

![image](https://github.com/saharmor/whisper-playground/assets/132049696/2cc4e929-42b3-40dc-b981-b4705044a4c7)

**Comment by ethanzrd** [comment]
Huh, that is odd. I'll test a bit with the recording tomorrow :)

**Comment by shanky100** [comment]
One more thing I checked , I am getting the transcript in my <p> tag but the issue is that it is not getting displayed on the web page. Here is a screenshot for the same.
Any guess/suggestion for this behaviour?

![image](https://github.com/saharmor/whisper-playground/assets/132049696/7b718521-2cc8-4132-9c33-eae888d56fe2)

**Comment by shanky100** [comment]
> @saharmor Thank you for the reply, I am attaching a screen recording which you can check, although in terminal I see the transcriptions after some time but I do not get nay print on the web page UI.
> 
>  issue.mp4 
> ![image](https://user-images.githubusercontent.com/132049696/262411101-c85624f9-c252-4e25-a657-bb1bd3b859a0.png)
> 
> I hope this helps to get a better understanding on the challenge I am facing.

is there any update/solution which can be recommended?

**Comment by ethanzrd** [comment]
As for the HTML problem, have you tried expanding the window/making the viewport larger?

I don't know what could be causing your transcription issue as your audio is valid, I'll try to run it on a clean Windows installation and get back to you.

**Comment by shanky100** [comment]
I made a fresh installation, and updated my browser and now it is working fine, although I had to made a change in the package.json , I updated "start": "react-scripts --openssl-legacy-provider start" to "start": "react-scripts start --openssl-legacy-provider"

Now I can see the transcripts being generated.

**Comment by shanky100** [comment]
I made a fresh installation and had to make change in package.json
"start": "react-scripts --openssl-legacy-provider start" to "start": "react-scripts start --openssl-legacy-provider"
Also I updated my chrome to latest version.

I can see the transcription now, but in starting for most of the segments the transcription being generated is empy, after 30 secs it starts to transcribe something but before that I get no transcript.
Is there any suggestion/solution which can be recommended.

![image](https://github.com/saharmor/whisper-playground/assets/132049696/803a4d50-eea3-4587-8dab-f233eb12528d)

**Comment by ethanzrd** [comment]
That's what I couldn't understand. I'm currently a bit busy with other things, but once some time frees up, I'll run a fresh Windows installation and run the project to see if there's any dependencies we might have missed that could be causing this strange behavior.
